src/Uno.UI/UI/Xaml/Media/SolidColorBrush.cs

```diff
@@ -127,5 +127,9 @@ public bool Equals(SolidColorBrush other)
 		public static bool operator ==(SolidColorBrush left, SolidColorBrush right) => Equals(left, right);
 
 		public static bool operator !=(SolidColorBrush left, SolidColorBrush right) => !Equals(left, right);
+
+		public static implicit operator Color(SolidColorBrush scb) => scb.Color;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'scb' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5444000)